easybuild/easyconfigs/d/dotNET-SDK/dotNET-SDK-8.0.302-linux-x64.eb

```diff
@@ -0,0 +1,36 @@
+##
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+#               Robert Mijakovic <robert.mijakovic@lxp.lu>
+#
+# License:: MIT
+# Notes::   requires glibc v2.14 or higher
+##
+easyblock = 'Tarball'
+
+name = 'dotNET-SDK'
```

Comment:
Please use dotNET-Core instead

Reply:
@Micket I changed the depencency to `dotNET-Core`.